shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-core/src/main/java/org/apache/shardingsphere/shadow/algorithm/shadow/column/ColumnRegexMatchShadowAlgorithm.java

```diff
@@ -81,11 +81,11 @@ private void checkOperation() {
     }
     
     @Override
-    public boolean isShadow(final Collection<String> shadowTableNames, final PreciseColumnShadowValue<String> shadowValue) {
+    public boolean isShadow(final Collection<String> shadowTableNames, final PreciseColumnShadowValue<Comparable<?>> shadowValue) {
         boolean containTable = shadowTableNames.contains(shadowValue.getLogicTableName());
```

Comment:
why containTable is not isContainTable or why others start with `is`

Reply:
Thanks.
Method `isShadow()` in the algorithm is used to determine whether the sql contains shadow coloring. 
Shadow tables and shadow algorithms are separately configured and referenced by ID. 
If the user customizes the algorithm, he needs to get the configured shadow tables to customize his own judgment algorithm. 
So the beginning of `isContainTable` is not used.